templates/zerver/help/zulip-on-android.md

```diff
@@ -0,0 +1,186 @@
+5. Write your message in the field at the bottom, the one that says *Tap on a
+   message to compose reply*.
+6. Hit the paper plane icon
+   (![Paper plane icon](/static/images/help/android-paperplane-icon.png)) to
+   send your message.
+
+![Stream message](/static/images/help/android-stream-message.png)
+
+### Sending a private message
+
+1. Press the pencil button at the right bottom side of the screen, in case the
+   compose box isn't already there.
+2. Make sure you are in the "stream message" mode. You should see the
+   icon of a megaphone
+   (![Megaphone icon](/static/images/help/android-megaphone-icon.png)) at the
+   right-hand side of the composing area.
+   If it isn't there, tap the person
+   (![Person icon](/static/images/help/android-person-icon.png)).
+3. Enter the email address of the person you want to chat with in the *Person*
+   field.
+
+!!! tip ""
+    Suggestions matching the entered email address will appear as you type it.
+
+4. Write your message in the field at the bottom, the one that says *Tap on a
+   message to compose reply*.
+5. Hit the paper plane icon
+  (![Paper plane icon](/static/images/help/android-paperplane-icon.png)) to
+  send your message.
+
+![Private message](/static/images/help/android-private-message.png)
+
+### Replying to a message
+
+1. Press the message you want to reply to in the message list.
+2. The *Stream* and *Topic* fields will automatically fill.
+3. Write your message in the field at the bottom, the one that says *Tap on a
+   message to compose reply*.
+4. Hit the paper plane icon
+   (![Paper plane icon](/static/images/help/android-paperplane-icon.png)) to
+   send your message.
+
+### Narrowing to a stream
+
+There are multiple ways of narrowing to a stream in Zulip's Android app.
+
+By swiping to the right from the border of the screen, or by hitting the menu
+(<i class="icon-vector-reorder"></i>) icon, you'll be able to see a list of the
+streams you're subscribed to.
+
+![Streams list](/static/images/help/android-streams-list.png)
+
+You can tap one of them to narrow to that stream.
+
+Another way is to press the name of a stream in a message. This will
+automatically narrow your message list to the pressed stream.
+
+### Narrowing to a topic
+
+You can narrow your message list to a specific topic by tapping the name of the
+topic over a message.
+
+### Searching for messages
+
+To look for messages containing some specific terms:
+
+1. Tap the magnet icon at the top of the screen.
+2. Write the desired search terms.
+3. Press enter in your keyboard.
+
+![Message search](/static/images/help/android-search.png)
+
+### Filtering messages
+
+You can filter the messages in your list. To do it:
+
+1. Press the filter icon in the top right side of the screen (between the magnet
+   and the three dots <i class="icon-vector-ellipsis-vertical"></i>).
+2. Choose the filtering criteria.
+3. The message list will refresh, showing only the matching Zulips.
```

Comment:
Change 'Zulips' to 'messages' in "showing only the matching Zulips". I don't think we establish anywhere that our messages are called Zulips.